includes/Dashboard.php

```diff
@@ -247,7 +247,7 @@ public function get_dashboard_settings() {
 				 * Issue: 1897
 				 * Creation date: 2020-05-21
 				 */
-				'enableAnimation'                 => true,
+				'enableAnimation'                 => false,
```

Comment:
Whoops, accidentally merged in flag set to true in my last animation PR 😱, putting it back to false.